### PR TITLE
Make Examples Pretty

### PR DESCRIPTION
**Pull request by NeelMohapatra**
- [x] Fix aspect ratio
- [x] Have to work on animating them
- [x] Make sure that they are responsive
- [x] Try to make showcase look better?
- [x] Fix Spacing in showcase examples
- [x] Remove `bgBackground` from `vl-examples.json`
- [x] Try to zoom into examples
- [x] Alter example properties in `vl-examples.json`

### CODE REVIEW

**Comment by domoritz** [comment]
I already did the showcase yesterday but it needs more examples.

**Comment by NeelMohapatra** [comment]
I know. But they are all justified. Do you want them in a block like Vega or is it okay if it remains that way?

**Comment by domoritz** [comment]
I like the way vega-lite looks now better.

**Comment by domoritz** [comment]
Later we might want to replace the logic for enumerating with jekyll code. It's odd that we need javascript to render the gallery. But that can come later.

**Comment by NeelMohapatra** [comment]
Open an issue for it and assign it to me

File: site/static/main.css [hunk 1]
```diff
@@ -484,7 +484,7 @@ footer .edit-page {
   -webkit-transition: background-position 2s;
 }
 .gallery .image:hover {
-  background-position: right bottom;
+  background-position: right bottom !important;
```
Comment (domoritz) [inline site/static/main.css]: No! Remove the background position from the examples instead.

Comment (NeelMohapatra) [inline site/static/main.css]: Yeah, I am working on it. That's what why I wanted to make it cleaner.

**Comment by domoritz** [comment]
Fantastic. One more thing before I merge. Can we use `backgroundSize` in the examples gallery? For some examples it will make sense fo zoom in a bit. And then let's delete all lines that have `backgroundPosition` in `vl-examples.json`. We don't need it anymore. 

Thanks @NeelMohapatra for the work! The examples look much nicer now and I love the new start page.

**Comment by domoritz** [comment]
Any idea why there is space on the homepage betweent eh examples now?

old:
screen shot 2017-04-16 at 00 00 25

new:
screen shot 2017-04-16 at 00 00 20

File: examples/vl-examples.json [hunk 2]
```diff
@@ -2,22 +2,15 @@
...
     },
     {
       "name": "bar_aggregate",
       "title": "Aggregate Bar Chart"
     },
     {
       "name": "bar_grouped",
-      "title": "Grouped Bar Chart",
-      "galleryParameters": {
-        "backgroundSize": "220%",
```
Comment (domoritz) [inline examples/vl-examples.json]: why did you delete this?

File: examples/vl-examples.json [hunk 3]
```diff
@@ -31,94 +24,66 @@
...
-        "backgroundSize": "120%",
-        "backgroundPosition": "50% 50%"
-      }
+      "title": "Histogram"
     },
     {
       "name": "area",
-      "title": "Area Chart",
-      "galleryParameters": {
-        "backgroundSize": "170%",
```
Comment (domoritz) [inline examples/vl-examples.json]: and this?

File: examples/vl-examples.json [hunk 4]
```diff
@@ -31,94 +24,66 @@
...
-        "backgroundPosition": "40% 15%"
-      }
+      "title": "Slope Graph"
     },
     {
       "name": "histogram",
-      "title": "Histogram",
-      "galleryParameters": {
-        "backgroundSize": "120%",
```
Comment (domoritz) [inline examples/vl-examples.json]: and this

File: examples/vl-examples.json (hunk 2 above)
Comment (domoritz) [inline examples/vl-examples.json]: I guess you looked through them and decided this

Comment (NeelMohapatra) [inline examples/vl-examples.json]: Yep. They looked bloated

File: examples/vl-examples.json (hunk 3 above)
Comment (NeelMohapatra) [inline examples/vl-examples.json]: Same reason. I don't think you need zoom here

File: examples/vl-examples.json (hunk 4 above)
Comment (NeelMohapatra) [inline examples/vl-examples.json]: Same reason
